src/vector/values/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Run as an agent daemonset
+role: "Agent"
+
+customConfig:
+  data_dir: /var/lib/vector
+  # Ensure e2e delivery of events
+  acknowledgements:
+    enabled: true
+  sources:
+    pod_logs:
+      type: "kubernetes_logs"
+      oldest_first: true
+    node_logs:
+      type: "file"
+      include: ["/var/log/*", "/var/log/kubernetes/**/*.log"]
+      oldest_first: true
+    internal_metrics:
+      type: internal_metrics
+
+  transforms:
+    pod_logs_labelled:
+      type: remap
+      inputs: ["pod_logs"]
+      source: |
+        if exists(.kubernetes.pod_labels."app.kubernetes.io/name") {
+          .app = .kubernetes.pod_labels."app.kubernetes.io/name"
+        } else if exists(.kubernetes.pod_labels.app) {
+          .app = .kubernetes.pod_labels.app
+        } else if exists(.kubernetes.pod_owner) {
+          .app = replace!(.kubernetes.pod_owner, r'^([^/]+/)', "")
+        } else {
+          .app = .kubernetes.pod_name
+        }
+
+        if exists(.kubernetes.pod_labels."app.kubernetes.io/component") {
+          .component = .kubernetes.pod_labels."app.kubernetes.io/component"
+        } else if !exists(.kubernetes.pod_labels.component) {
+          .component = .kubernetes.pod_labels.component
+        } else {
+          .component = ""
+        }
+
+    node_logs_labelled:
+      type: remap
+      inputs: ["node_logs"]
+      source: |
+        .node_name = "${NODE_HOSTNAME}"
+        if contains(string!(.file), "/var/log/kubernetes/") {
+          .job = "kubernetes-logs"
+        } else {
+          .job = "varlogs"
+        }
```

Comment:
This might be the 'recommended way' but curious if there is an easy way to just have separate sources for each `.job` type instead of running this VRL one every line?

Reply:
We could remove the VRL here, but I think it's more DRY to do it this way. Without the VRL you need:
- 2 sources: ~3 lines duplicated
- 2 sinks: ~14 lines duplicated

It was really the duplication of the sinks that pushed me towards the remap with VRL - since labels cannot be added on the file source itself you end up needing 2 sinks if you want different labels from the two different sources. Are there particular concerns with this path? I can split out into 2 sources and sinks if desired, but wasn't overly concerned with the addition of this check based on upstream [VRL performance notes](https://vector.dev/docs/reference/vrl/#performance). Happy to chat more and see what makes the most sense here.

Reply:
I think another alternative - we could just rethink the job label here and use `varlogs` for both, and lean into Loki querying for someone to identify specific k8s audit logs. That would be a slight shift and probably good to callout, but it would remove the VRL usage and keep this DRY.

Reply:
Hmm.  I think based on that I'm good with just leaving it.  If it becomes an issue we can revisit but you're probably right in that it will be fine performance wise.